src/plugins/intel_gpu/src/graph/graph_optimizer/mark_runtime_skippable_nodes.cpp

```diff
@@ -19,9 +19,12 @@
 #include "scatter_nd_update_inst.h"
 #include "program_helpers.h"
 
+#include <unordered_map>
+
 using namespace cldnn;
 
 void mark_runtime_skippable_nodes::run(program& p) {
+    std::unordered_map<primitive_id, uint8_t> runtime_skippable_depth;
```

Comment:
Can we use node ptr instead of string here?

Reply:
Updated it! Thanks!